docs/pages/agents/introduction.mdx

```diff
@@ -14,7 +14,7 @@ your infrastructure.
 
 ### Services
 
-Each Teleport agent can run one or more services.  A Teleport instance runs a
+Each Teleport agent can run one or more **services**. A Teleport instance runs a
```

Comment:
```suggestion
Each Teleport process can run one or more **services**. A Teleport instance runs a
```

We often say things like "where is your DB agent running?" - to which the answer can be "in the same process as the proxy service".